titus-server-master/src/test/java/com/netflix/titus/master/integration/v3/job/JobSubmitAndControlNegativeTest.java

```diff
@@ -174,6 +174,7 @@ public void testJobWithoutImage() throws Exception {
     @Test
     @Ignore("Until we support digests")
     public void testJobWithBothTagAndDigest() throws Exception {
+        // TODO(Andrew L): Enable this test.
```

Comment:
Was this an accident or do you want to do this in another PR?

Reply:
Accident. I'll remove. There shouldn't be any other changes to that file.